script/hassfest/quality_scale.py

```diff
@@ -1089,6 +1089,7 @@ class Rule:
     "v2c",
     "vallox",
     "vasttrafik",
+    "vegehub",
```

Comment:
Please add a quality scale file instead. New integrations require Bronze tier:

https://developers.home-assistant.io/docs/core/integration-quality-scale/

https://developers.home-assistant.io/docs/core/integration-quality-scale/#keeping-track-of-the-implemented-rules